src/graphql/__tests__/entries.test.ts

```diff
@@ -18,12 +19,14 @@ describe('Update GraphQL Entry', () => {
     data,
     update = entry as EntryProps,
     locale = EN,
+    entityReferenceMap = new Map<string, EntryProps | AssetProps>(),
```

Comment:
do we maybe want to extract the type of entityReferenceMap and add it to `types.ts`?

Reply:
Good idea! done